### PR TITLE
Added rustdoc for `hsmtool` and `ot_certs` crates to the site

### PR DESCRIPTION
**Pull request by HU90m**

### CODE REVIEW

**Review by jwnrt — APPROVED** [review]
Thanks @HU90m

File: sw/host/README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
 # Host Software
 
-- [Generated documentation for `opentitanlib`](https://opentitan.org/gen/rustdoc/opentitanlib)
+- [OpenTitanLib](./opentitanlib/README.md)
+- [OpenTitanTool](./opentitantool/README.md)
+- [OpenTitanSession](./opentitansession/README.md)
+- [OpenTitan Certificate Generator](./ot_certs/README.md)
+- [Hardware Security Module (HSM) tool](./hsmtool/README.md)
+- [TPM2 Test Server](./tpm2_test_server/README.md)
```
Comment (hcallahan-lowrisc) [inline sw/host/README.md:8]: WDYT about re-ordering this list to match the order in the SUMMARY.md navbar?

File: sw/host/ot_certs/README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # OpenTitan certificate generator
 
+*[Click here for the generated documentation of `ot_certs`.](https://opentitan.org/gen/rustdoc/to_certs)*
```
Comment (hcallahan-lowrisc) [inline sw/host/ot_certs/README.md:3]:
```suggestion
*[Click here for the generated documentation of `ot_certs`.](https://opentitan.org/gen/rustdoc/ot_certs)*
```

File: util/site/build-docs.sh [hunk 3]
```diff
@@ -157,10 +157,12 @@ buildSite () {
...
-    target_rustdoc="sw/host/opentitanlib:opentitanlib_doc"
-    target_rustdoc_output_path="${bazel_out}/k8-fastbuild/bin/$(echo ${target_rustdoc} | tr ':' '/').rustdoc" #TODO : get the target's path using cquery
-    ./bazelisk.sh build --experimental_convenience_symlinks=ignore "${target_rustdoc}"
-    cp -rf "${target_rustdoc_output_path}"/* "${rustdoc_dir}"
+    for target_rustdoc in "sw/host/opentitanlib:opentitanlib_doc" "sw/host/hsmtool:hsmlib_doc" "sw/host/ot_certs:ot_certs_doc"
+    do
+      target_rustdoc_output_path="${bazel_out}/k8-fastbuild/bin/$(echo ${target_rustdoc} | tr ':' '/').rustdoc" #TODO : get the target's path using cquery
+      ./bazelisk.sh build --experimental_convenience_symlinks=ignore "${target_rustdoc}"
+      cp -rf "${target_rustdoc_output_path}"/* "${rustdoc_dir}"
+    done
```
Comment (hcallahan-lowrisc) [inline util/site/build-docs.sh:165]: This should fix the failing build in CI. It fails for me locally too in the same way.
```suggestion
      cp -rf "${target_rustdoc_output_path}"/* "${rustdoc_dir}"
      chown -R $USER: "${rustdoc_dir}"
      chmod -R +w "${rustdoc_dir}"
    done
```

**Review by hcallahan-lowrisc — APPROVED** [review]
Looks good, thanks @HU90m  :)

File: sw/host/README.md (hunk 1 above)
Comment (HU90m) [reply sw/host/README.md:8]: That's a good idea. Changed the order in the navbar

**Pull request merged**